src/components/NewPost/NewPost.js

```diff
@@ -16,9 +18,17 @@ const NewPost = () => {
 
   const submitHandler = (event) => {
     event.preventDefault();
-    console.log(inputState);
+    picturesData.push(inputState);
+    alert('Zdjęcie zostało dodane.');
+    setInputState(() => ({
+      id: '',
```

Comment:
Na drugi rzut oka, jeden detal: zwróć uwagę, że tworząc nowy stan wyżej, przypisałaś id (new Date().getTime()), a tutaj id jest puste. Zwróć uwagę, że w obu przypadkach ustawiasz tam obiekty, więc może warto by było stworzyć jakiś domyślny initialState o określonych parametrach, a potem wykorzystywać go w obu miejscach, zamiast dwa razy tworzyć obiekt i ryzykować, że gdy coś się zmieni - zmienisz to tylko w jednym z tych miejsc? :)

Reply:
No tak :) Zrobione 👍